### PR TITLE
- fix external args tpl

### PR DESCRIPTION
**Pull request by vwaurich**

### CODE REVIEW

File: Compiler/Template/CodegenCFunctions.tpl [hunk 1]
```diff
@@ -2355,7 +2355,7 @@ template extArg(SimExtArg extArg, Text &preExp, Text &varDecls, Text &auxFunctio
     '(<%extType(t,isInput,true)%>) <%name%>_c89'
   case SIMEXTARG(cref=c, isInput=ii, outputIndex=0, type_=t) then
     let cr = match t case T_STRING(__) then contextCref(c,contextFunction,&auxFunction) else extVarName(c)
-    (match t case T_STRING(__) then 'MMC_STRINGDATA(<%cr%>)' else cr)
+    (match t case T_STRING(__) then '<%cr%>' else cr)
```
Comment (sjoelund) [inline Compiler/Template/CodegenCFunctions.tpl]: This just says if string then cr else cr. So just use cr, right?

**Comment by OpenModelica-Hudson** [comment]
The job failed to compile; for details, see [OpenModelica_TEST_PULL_REQUEST 2016-09-30_08-46-43](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3168/console).

File: Compiler/Template/CodegenCFunctions.tpl (hunk 1 above)
Comment (vwaurich) [inline Compiler/Template/CodegenCFunctions.tpl]: True.

**Comment by OpenModelica-Hudson** [comment]
The job failed to compile; for details, see [OpenModelica_TEST_PULL_REQUEST 2016-09-30_20-05-34](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3170/console).

**Pull request closed without merging**